core/store/src/trie/state_parts.rs

```diff
@@ -14,6 +14,18 @@ use crate::{PartialStorage, StorageError, Trie, TrieChanges, TrieIterator};
 use near_primitives::contract::ContractCode;
 use near_primitives::state_record::is_contract_code_key;
 
+// to specify a part we always specify both part_id and num_parts together
```

Comment:
here too

Reply:
@mzhangmzz this `core` package is not importing any other `crate` which implements `PartId` . Take a look at it's [`cargo.toml` file](https://github.com/near/nearcore/blob/master/core/store/Cargo.toml#L30). Which `crate` should I add here to use the `PartId` struct ?